package.json

```diff
@@ -46,77 +46,77 @@
   },
   "dependencies": {
     "csstype": "^2.6.0",
-    "deep-object-diff": "^1.0.4",
+    "deep-object-diff": "^1.1.0",
     "emotion": "^8.0.8",
-    "history": "^4.6.1",
-    "lodash": "^4.17.4",
-    "marksy": "^6.0.2",
-    "normalize.css": "^7.0.0",
-    "prismjs": "1.6.0",
+    "history": "^4.7.2",
+    "lodash": "^4.17.11",
+    "marksy": "^6.1.0",
+    "normalize.css": "^8.0.1",
+    "prismjs": "1.15.0",
```

Comment:
**WARNING**: `prismjs` has to match whatever is in `react-live` **exactly** or horrible things happen. (Also, if any other transitive dep brings `prismjs` in that _also_ has to match).

See, e.g. #546 #558 #516